src/components/itemBox/ItemBox.tsx

```diff
@@ -24,7 +26,31 @@ export class ItemBox extends React.Component<IItemBoxProps, any> {
         tooltip: {
             title: '',
         },
+        highlight: '',
     };
+    private el: any;
+
+    componentDidUpdate(prevProps: IItemBoxProps, prevState: IItemBoxProps) {
+        if (prevProps.active !== this.props.active && this.props.active) {

```

Comment:
Can't this be simplified as `if (!this.props.active && this.props.active) {`?

Reply:
did you mean `if (!this.prevProps.active && this.props.active)`

Reply:
Isn't `!this.props.active && this.props.active` always gonna be false?